scala3/new-in-scala3.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: singlepage-overview
+title: That's new in Scala 3
+---
+The upcoming, exciting new version of Scala 3 brings many improvements and
+new features. Here we provide you with a quick overview of the most important
+changes. If you want to dig deeper, there are a few references at your disposal:
+
+- The [Scala 3 Book]({% link _overviews/scala3-book/introduction.md %}) targets developers new to the Scala language.
+- The [Syntax Summary][syntax-summary] provides you with a formal description of the new syntax.
+- The [Language Reference][reference] gives a detailed description of the changes from Scala 2 to Scala 3.
+- The [Migration Guide][migration] provides you with all of the information, necessary to move from Scala 2 to Scala 3.
+
+## That's new in Scala 3
```

Comment:
This is a nice list! I think it would be good to have @odersky’s opinion about it, to make sure we don’t forget anything.

Reply:
Maybe we can add a section about OOP features: trait parameters, open classes, export clauses, safe-initialization?

Reply:
Yes! Great idea, I'll compile a list for that. Maybe "Re-envisioned: Object-Oriented Programming"